.gitignore

```diff
@@ -1 +1,5 @@
-vendor
+vendor/
+composer.lock
```

Comment:
Why do you ignore the composer.lock?

Reply:
Because this is not a stand-alone application, but a library (module) to be used by other applications.

The lock file will not have any effect on projects which have this module as a dependency (luckily as it would quickly become a dependency hell otherwise)

So, while it shouldn't cause necessarily cause any issue, it is considered best practice *not* to commit the `composer.lock` file for libraries.
Bit like `Gemfiles` for Gems.

Refs:
* https://blog.martinhujer.cz/17-tips-for-using-composer-efficiently/ (tip 6)
* https://getcomposer.org/doc/02-libraries.md#lock-file

Reply:
Got it. Thanks!